WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/PointOfSaleCardPresentPaymentTapSwipeInsertCardMessageView.swift

```diff
@@ -4,15 +4,10 @@ struct PointOfSaleCardPresentPaymentTapSwipeInsertCardMessageView: View {
     let viewModel: PointOfSaleCardPresentPaymentTapSwipeInsertCardMessageViewModel
 
     var body: some View {
-        HStack {
-            VStack {
-                Text(viewModel.title)
-                Text(viewModel.message)
-            }
-
-            Button(viewModel.cancelButtonViewModel.title,
-                   action: viewModel.cancelButtonViewModel.actionHandler)
-        }
+        POSCardPresentPaymentMessageView(viewModel: .init(imageName: "pos_ready_for_payment",
```

Comment:
❓ Should we move the `pos_ready_for_payment` string to some sort of extension like we do for `UIImage+Woo`?